components/dotcms/layout/Nav/Nav.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { useRouter } from 'next/router';
+
+import LanguageSelector from '../LanguageSelector';
+import MenuList from '../../layout/Nav/MenuList';
+import PageContext from '../../../../contexts/PageContext';
+import SocialMediaMenu from '../../layout/Nav/SocialMediaMenu';
+import logo from '../../../../public/logo.png';
+import menuIcon from '../../../../public/menu.svg';
+import useNav from '../../../../hooks/useNav';
+import { MainNav, NavMenu } from '../../../../styles/nav/nav.styles';
+import RouterLink from '../../../RouterLink';
+import { getCurrentLanguage } from '../../../../utilities/dotcms/locale';
+
+export default function Nav() {
+    const nav = useNav();
+    const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+    const {
+        languageProps: { defaultLanguage }
+    } = useContext(PageContext);
+
+    const handleOpenMenu = (e) => {
+        e.preventDefault();
+        setIsMenuOpen(!isMenuOpen);
+    };
+
+    return (
+        <div className="container">
+            <MainNav className="main-nav">
+                <RouterLink
+                    href={
+                        getCurrentLanguage() && !defaultLanguage ? `/${getCurrentLanguage()}` : `/`
```

Comment:
I don't think we have do pass the language here because we have it in the `RouterLink` no?

Reply:
This is just for the logo.